main.py

```diff
@@ -39,9 +39,16 @@ def main():
 	config.read_string(fix_config("config/config.ini"))
 
 	defaults = config["DEFAULT"]
-
+	render_pass_file = defaults.get("RenderPassFile")
+								 
 	if defaults.getboolean("UseExampleRenderPasses"):
 		config.read_string(fix_config("config/example.renderpasses.ini"))
+	elif render_pass_file:
```

Comment:
This condition seems suspicious.
Just an "else" is fine in this case.

Reply:
If RenderPassFile is not defined in one of the configs then `render_pass_file == None`. Using an `else` causes an error a few lines later if UseExampleRenderPasses is also set to `no` or commented out when it tests to see if the file exists.

Agree, it is a bit of a corner case, but would rather err on the side of trapping errors and giving meaningful feedback than letting it bomb.

Reply:
It's fine to print out an error in this case since if the user doesn't use example render passes then they should have their own. Now it will run the script without any render passes and it won't print any error about not having any which is really weird behavior for the user.

Reply:
Before this PR, if a user has no renderpasses in either the examplerenderpasses or config files, then it drops through just printing DONE!  So, this PR introduces no new behaviour. Agree that itbe better that zero renderpasses are flagged in all situations.